recipes/qtmolecularnetwork/meta.yaml

```diff
@@ -0,0 +1,105 @@
+{% set version = "0.6.1" %}
+
+package:
+    name: qtmolecularnetwork
+    version: {{ version }}
+
+source:
+    url: https://github.com/metgem/QtMolecularNetwork/archive/v{{ version }}.tar.gz
+    sha256: 455329bbf0d2223637012181de6ff8b3fe9e64cb204f0c934df8a59a2a0f9692
+    patches:
+        - 0001-libcxx.patch                   # [linux]
+
+build:
+    number: 0
+    missing_dso_whitelist:
+        - $RPATH/libRDKitDepictor.1.dylib     # [osx]
+        - $RPATH/libRDKitMolDraw2D.1.dylib    # [osx]
+        - $RPATH/libRDKitSmilesParse.1.dylib  # [osx]
+        - $RPATH/libRDKitRDInchiLib.1.dylib   # [osx]
+        - $RPATH/libRDKitDepictor.so.1        # [linux]
+        - $RPATH/libRDKitMolDraw2D.so.1       # [linux]
+        - $RPATH/libRDKitSmilesParse.so.1     # [linux]
+        - $RPATH/libRDKitRDInchiLib.so.1      # [linux]
+
+requirements:
+    build:
+        - python {{ python }}
+        - cmake
+        - {{ compiler('cxx') }}
+        - make                                # [not win]
+        - rdkit-dev
+        - eigen
+        - mesalib                             # [unix]
+        - make                                # [not win]
+    host:
+        - qt-main >=5.15
+    run:
+        - qt-main >=5.15
+        - rdkit
```

Comment:
So, do I need to freeze rdkit version? `- rdkit ==2022.03.3`

Reply:
No. I would do something like this:

```yaml
requirements:
  host:
   - rdkit-dev
   - rdkit
  run:
    - {{ pin_compatible(rdkit, max_pin='x.x.x') }}
```
Because then this recipe is always build against the latest version.

Reply:
I now have a better understanding of the current rdkit. It is empty on unix and contains headers (and a library?) on Windows? I saw you have an open PR to try and fix the rdkit feedstock.

Reply:
rdkit-dev was initially created to contains all .lib for static linking but the building process change recently and does not include .lib files anymore.